apps/session-manager/src/utils/responses.ts

```diff
@@ -74,3 +77,39 @@ export const unhandledResponseStatus = (
   status: number,
 ): IResponseErrorInternal =>
   ResponseErrorInternal(`unhandled API response status [${status}]`);
+
+/**
+ * Interface for 401 unauthorized
+ */
+export interface IResponseErrorUnauthorized
```

Comment:
This is a reusable logic cross project, this type of response is missing in `ts-commons`. What you think if we create a new package in this monorepo to extend the responses declared on `ts-commons` lib?
Otherwise this response should be added into the `ts-commons`

Reply:
I think it would be better to put it directly inside `ts-commons`. [I created a PR inside ts-commons repo](https://github.com/pagopa/io-ts-commons/pull/423)

Reply:
updated the dependency in https://github.com/pagopa/io-auth-n-identity-domain/pull/18/commits/8a083e326d241baa8febfd77f182b4588be698bc